### PR TITLE
chore: update build command

### PR DESCRIPTION
**Pull request by harish-sethuraman**
use `build:*` and `--parallel`

### CODE REVIEW

**Comment by harish-sethuraman** [comment]
`build:*` is same but --parallel is new

**Comment by snitin315** [comment]
We need to build sass before building eleventy. See deploy preview its broken.

File: package.json [hunk 1]
```diff
@@ -17,7 +17,7 @@
         "build:webpack": "webpack --mode=production",
         "prestart": "npm run build",
         "start": "node tools/dev-server.js",
-        "build": "npm-run-all install:playground build:sass build:eleventy build:webpack images",
+        "build": "npm-run-all build:sass --parallel install:playground build:eleventy build:webpack images",
```
Comment (mdjermanovic) [inline package.json:20]: Shouldn't `build:webpack` wait for` install:playground` to finish?

Comment (mdjermanovic) [inline package.json:20]: Shouldn't `build:webpack` wait for` install:playground` to finish?

Comment (mdjermanovic) [reply package.json:20]: Also, `images` script works directly on `_site/assets/images`, so it seems that it should wait for `build:eleventy` to copy the images there (i.e., to finish).

Comment (nzakas) [reply package.json:20]: Yes, I don’t think we can safely do these in parallel. Nice idea, just doesn’t work in this case.

**Comment by nzakas** [comment]
@harish-sethuraman i think we are in the same spot on this PR as we are in the new website one. Can you take another look?

**Comment by harish-sethuraman** [comment]
https://github.com/eslint/playground/blob/3b54c11b88ba1afe583f8fd0283816ac6a9a94b3/tools/dev-server.js#L153-L155

Yea we can close this but before that can we confirm if we really want to add `build:sass` command to the development server as in other repos? also I see that we miss the images command too. Thoughts?

**Comment by nzakas** [comment]
@snitin315 ?

**Comment by snitin315** [comment]
In other repositories, I see we are [building sass and then again watching sass](https://github.com/eslint/eslint/blob/db387a87b23c398ed691ba35bdf3b6aa7dcfe750/docs/package.json#L17)

So if `--watch` also does the initial build(which I feel it does) then I suppose we are fine with `watch:sass` only in dev-server.

**Comment by harish-sethuraman** [comment]
- https://github.com/eslint/new.eslint.org/pull/223#discussion_r887395415


Context:
Screenshot 2022-06-21 at 11 20 54 AM

**Comment by snitin315** [comment]
I see, then let's add `build:sass` to dev server as well.

**Comment by harish-sethuraman** [comment]
We won't need prestart which eventually triggers the complete build process. We only need to watch the sass, eleventy and webpack. Build is not required for dev server. It is consuming time for starting up.

**Comment by harish-sethuraman** [comment]
npm is down. Would need a re trigger once it comes up.

**Review by nzakas — APPROVED** [review]

**Comment by nzakas** [comment]
Just retriggering the deploy preview to make sure nothing is broken.

**Comment by harish-sethuraman** [comment]
bumping for merge :)

**Pull request merged**